Makefile

```diff
@@ -1079,3 +1079,16 @@ endif
 	@sudo cp -vf $(iso) /var/lib/tftpboot/theseus/
 	@sudo systemctl restart isc-dhcp-server 
 	@sudo systemctl restart tftpd-hpa
+
+test: export override QEMU_FLAGS += -device isa-debug-exit,iobase=0xf4,iosize=0x04
+test: export override QEMU_FLAGS += -nographic
+test: export override FEATURES +=--features theseus_tests
+test: export override FEATURES +=--features qemu_test
+test: export override RUSTFLAGS +=--cfg qemu_test
```

Comment:
The only reason I didn't do that is because I don't think the feature could be considered "additive", but yea it's probably better than yet another `cfg`.